apps/librelingo_yaml_loader/librelingo_yaml_loader/yaml_loader.py

```diff
@@ -380,6 +382,18 @@ def _load_module(path: str, course: Course):
     """
     filepath = Path(path) / "module.yaml"
     data = _load_yaml(filepath)
+
+    schemapath = "../../apps/librelingo_yaml_loader/schemas/module.json"
+    with open(schemapath) as fp:
+        schema = json.load(fp)
+
+    try:
+        jsonschema.validate(data, schema)
+    except jsonschema.ValidationError as validation_error:
+        raise RuntimeError(
+            f"There is an error with the schema at the following file path: {filepath}"
```

Comment:
will this hide the particular details of the validation error? isn't it better to simply let the whole process fail at this point? (so simply removing the try-except?)